client/src/components/NavBar/AuthBars/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,71 @@
+import Button from '@material-ui/core/Button';
+import { Box, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import Notification from './Notification/Notification';
+import { useState, useEffect } from 'react';
+import getUnread from '../../../../helpers/APICalls/getAllUnreadNotifications';
+
+interface notification {
+  type: string;
+  title: string;
+  description: string;
+  read: boolean;
+  date: Date;
+  image: string;
+}
+
+const Notifications = (): JSX.Element => {
+  const classes = useStyles();
+  const [notifications, setNotifications] = useState<notification[]>([
+    {
+      type: '',
+      title: 'filler',
+      description: '',
+      read: false,
+      date: new Date('1/1/2000'),
+      image: '',
+    },
+  ]);
+
+  useEffect(() => {
+    getUnread().then((data) => {
+      const notificationsData = data.success?.notifications;
+      const notificationsArray: notification[] = [
+        {
+          type: '',
+          title: '',
+          description: '',
+          read: false,
+          date: new Date('1/1/2000'),
+          image: '',
+        },
+      ];
+      notificationsData?.forEach((n) => notificationsArray.push(n));
+      notificationsArray.shift();
+      setNotifications(notificationsArray);
+    });
+  }, []);
+
+  return (
+    <Box className={classes.notificationContainer}>
+      {notifications.map((notification) => {
+        return (
+          <Notification
+            key={notifications.indexOf(notification)}
```

Comment:
Please avoid usage of indexes as keys in React

Ref: https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9